tests/test_devices.py

```diff
@@ -623,6 +798,38 @@ def test_device_update_state_machine_disarmed(client, mocker):
                 2: {"id": 3, "index": 2, "element": 3, "excluded": True, "status": False, "name": "Outdoor Sensor 2"},
             },
         },
+        {
+            "last_id": 3,
+            "outputs": {
+                0: {
+                    "id": 1,
+                    "index": 0,
+                    "element": 1,
+                    "controldeniedtousers": False,
```

Comment:
Is this the way it's written? it should be in snake_case, exactly as the library is sending back to us.